vendor/github.com/jacobsa/gcloud/gcs/read.go

```diff
@@ -138,6 +153,38 @@ func (b *bucket) NewReader(
 	return
 }
 
+
+// Custom function made to create a new reader using Storage Client Library.
+func NewReaderSCL(
+        ctx context.Context,
+        req *ReadObjectRequest, bucketName string) (rc io.ReadCloser, err error){
+	// Create a client if there is not one already created.
+        if sclClient == nil {
+		fmt.Println("Client Created")
+		sclClient, err = storage.NewClient(ctx)
```

Comment:
.Close() at some point on this?

Reply:
But if we will close, then we will need to create the client again and in the documentation it was mentioned that we should reuse the client as much as possible.